dfs/nfs/mounts_test.go

```diff
@@ -11,7 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// +build unit
+// +build unit, linux
```

Comment:
You have a couple of test files where you've limited it to linux. Shouldn't we provide _darwin versions of these? (I'll admit it makes it easier to pass on Mac OS. ;-) ) Maybe at least open a ticket to come back to this later...